java/src/jmri/jmrix/lenz/XNetTurnout.java

```diff
@@ -132,7 +134,7 @@ public XNetTurnout(String prefix, int pNumber, XNetTrafficController controller)
         _prefix = prefix;
         mNumber = pNumber;
 
-        requestList = new LinkedBlockingQueue<>();
+        requestList = new ArrayDeque<>();
```

Comment:
Why a Deque here?  we really only need a Queue.  Seems like extra overhead.

Reply:
> I am particularly concerned that code appears to have been removed that triggers the OFF messages for turnout based on feedback to turnouts.

There should not be any removals - at least not intentional ones. This PR is *not* intended to change the behaviour in any way; if it does, it's a bug that must be fixed.

Reply:
Good, let's have a`LinkedList`, which is probably the simplest implementation of Queue. However its runtime overhead is higher than `ArrayDeque` backed by an array.

Reply:
OK.  That is a reason.  I just asked why, didn't ask for a change.

Reply:
@pabender I do value code quality and correctness: the field ought to be `Queue`. There was no irony in my answers -- I do need to change my bad habits when a reviewer discovers them.